2016-08/august-05.md

```diff
@@ -0,0 +1,73 @@
+
+* It was brought up that we need to create a feasible action plan that we can share with the Dan Abramov's, Kent Dodds', and Patrick Stapleton's of the JS community to pass along and promote. It is in everyones best interest to work together to create documentation and having a strategy to accomplish this is vital. We spent time listing out the immediate MVP documents that would the team agreed could allow us to life the webpack 2.x beta status. 
+
+* Juho created a [Github Milestone for all these tasks that we can give to people](https://github.com/webpack/webpack.io/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Webpack+2+-+Documentation+MVP%22)!!!! Barring any major or breaking changes from popular community loaders or plugins (as well as the core loaders and plugins), once this is finished, we believe this is our 'go-ahead' to lift the beta status. We all agreed that although this wouldn't contain all of the documentation we aspired to tackle, that this would give the (first time) users _that need it the most_, the opportunity to have a _far more_ augmented learning experience than what is available. 
+
+* Everyone also agreed that we could technically accomplish this by the end of August. **Yes people, it's getting more than real, this is an estimated timeline of webpack 2.x being out of beta!** :heart_eyes: :heart_eyes: :heart_eyes:. If all goes well and we meet our deadline, then we can lay out our next milestone for more advanced "revisions" of our existing documentation migrated over to the `webpack/webpack.io` repo.
+
+* **We will want/need lots of help!!!** So please share the link of the milestone above. The milestone will also include general guidelines for how to author one of these docs. And once we have a collection of people we can meet weekly via hangout, etc. to track and manage the Milestone goals, etc. There is also a [getting started page](https://github.com/webpack/webpack.io/blob/develop/content/get-started.md) we have referenced on the repo as well for those worried about contributing. It doesn't have to be perfect the first time around. 
+
+#### Publication
+
+* Sean admitted he has slacked a little bit on getting a new article for our publication, but wants to make it a priority to announce that the angular-cli and react-create-app are both powered by webpack now. 
+
+* Its a huge milestone of integration for us. Including Laravel and Rails:Middleman and Grails new asset Pipline. The list of webpack users is seeming to continue to increase and the learning and constructive criticism continues to grow. This _is the tool you made and helped customize_ in the end, so every person, framework, library, boilerplate, MVC, that uses webpack is another win for you and the webpack team. 
```

Comment:
As a long-time webpack user who kind of bet the farm on webpack, seeing the widespread adoption has been really exciting!